### DISCUSSION TITLE
can't install

### DISCUSSION THREAD

**Discussion by yuchen1204** [coder/code-server#5381] [Q&A]
![Screenshot_20220726-173430](https://user-images.githubusercontent.com/92127505/180976107-cd16781d-a3ab-4411-8b84-9a69087665a3.png)
log[Log](http://148.100.78.194/2022-07-26T09_24_56_178Z-debug.log)

**Comment by lg-nineoneninethreenine** [comment]
My mind be like: bro seriously

Logic goes ticking: your node version is 10, when we need 16, so try finding guides to update node on Termux(I saw that!) since normal methods result in missing deps.

**Reply by lg-nineoneninethreenine** [reply]
ohcrap

It's Argon2 again!?

**Reply by jsjoeio** [reply]
darn! is the issue with argon2 still happening?

**Comment by lg-nineoneninethreenine** [comment]
This seems to be a bug again from #4422, we need to know _more_.
